distros/kubernetes/nvsentinel/values-tilt.yaml

```diff
@@ -15,6 +15,7 @@
 global:
   dryRun: false
   kubeVersion: 1.31.0
+  installMongoDB: true
```

Comment:
This is not needed with the recent changes on main. This is likely a remanent of a bad merge, can you please check?